tuneme/forms.py

```diff
@@ -0,0 +1,38 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+
+class LoginForm(forms.Form):
+    username = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Username'),
+    }))
+    password = forms.CharField(widget=forms.PasswordInput(attrs={
+        'placeholder': _('4 Digit PIN'),
+    }))
+
+
+class RegistrationForm(forms.Form):
+    username = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Username'),
+    }))
+    password = forms.CharField(widget=forms.PasswordInput(attrs={
+        'placeholder': _('4 Digit PIN'),
+    }))
+
+
+class EditProfileForm(forms.Form):
+    alias = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Display Name'),
+    }))
+
+
+class ProfilePasswordChangeForm(forms.Form):
```

Comment:
@saeedmb8 you idea is correct, but the implementation is missing this:
In the imports you should add
```python
from molo.profiles import forms as profile_forms
```
and here this line should change to 
```python
class ProfilePasswordChangeForm(profile_forms.ProfilePasswordChangeForm):
```
Otherwise you're not inheriting from the original forms.